Samples/Windowing/cs-winui/MainPage.xaml

```diff
@@ -14,7 +14,7 @@
             <RowDefinition Height="Auto"/>
             <RowDefinition />
         </Grid.RowDefinitions>
-        <Grid Grid.Row="0" x:Name="MyTitleBar" Background="Blue" x:FieldModifier="public" Visibility="Collapsed">
+        <Grid Grid.Row="0" x:Name="MyTitleBar" Background="AliceBlue" x:FieldModifier="public" Visibility="Collapsed">
```

Comment:
Why a change of color in this iteration of the sample? Accessibility issue with Blue?

Reply:
I wanted to avoid the loudness of the blue. We are moving towards more subtle theming.